src/app/ui/export_file_window.cpp

```diff
@@ -198,13 +208,24 @@ void ExportFileWindow::updateAniDir()
     SelectedFrames selFrames;
     Tag* tag = calculate_selected_frames(
       UIContext::instance()->activeSite(), framesValue, selFrames);
-    if (tag)
-      anidir()->setSelectedItemIndex(int(tag->aniDir()));
+    if (tag) {
+      if (isPlaySubtags())
+        anidir()->setSelectedItemIndex(int(doc::AniDir::FORWARD));
+      else
+        anidir()->setSelectedItemIndex(int(tag->aniDir()));
+    }
```

Comment:
This was my reasoning: When the user checks the play subtags option then the direction of the animation is given by the tag properties (because I thought that the user is checking the option to actually follow what the tag says), then the "animation direction" combobox will change the whole animation direction.
Also I thought in this scenario: for instance, suppose I have a tag that has a "reverse" direction, then I select it and the UI sets the anidir to Reverse. Then when I check the "play subtags" option I would like that the animation respects what my tags say (the thing is that a tag can algo have subtags inside, so at the end playing all frames is the same as playing one tag that contains subtags), then if the animation direction is kept to reverse, it will reverse the reverse of the tag...

Reply:
But yes, you are right that it is a bug to my reasoning that the anidir is not being set to forward when checking the play subtags checkbox and the anidir was set in a different value.

Reply:
In summary, my intention was to always make Animation Direction work in such a way that it sets the direction of the final animation.

Reply:
From an user point of view, I think they will never need to click/unclick the Play subtags checkbox. I forgot to mention to change the `<option id="play_subtags" type="bool" default="false" />` to `default="true"` because this is the expected behavior by users and it's the fix that they are requesting.

As the Play subtags should be checked by default (expected behavior for user that set the Repeat field of a tag, etc.), it makes sense that the anidir should show the tag anidir when we export specifically one tag (which is the current UI behavior of v1.3.2). The "forward" item to indicate "export the tag in its anidir but we'll not touch the final result keeping it going forward the whole result" is confusing because it's too internal to the implementation)

Reply:
> From an user point of view, I think they will never need to click/unclick the Play subtags checkbox. I forgot to mention to change the `<option id="play_subtags" type="bool" default="false" />` to `default="true"` because this is the expected behavior by users and it's the fix that they are requesting.

👍 

> As the Play subtags should be checked by default (expected behavior for user that set the Repeat field of a tag, etc.), it makes sense that the anidir should show the tag anidir when we export specifically one tag (which is the current UI behavior of v1.3.2). The "forward" item to indicate "export the tag in its anidir but we'll not touch the final result keeping it going forward the whole result" is confusing because it's too internal to the implementation)

Let's say we have something like this, where we have a tag (with reverse direction) that is parent of two other tags and extends to the whole animation ([tetstags.aseprite.zip](https://github.com/aseprite/aseprite/files/14104150/tetstags.aseprite.zip))
Then if the user selects the "parent tag" we should show the "Reverse" in the anidir combo, right? Then when exporting the animation it should follow the selected direction as the tag says.
But, in the other case, when the user selects "All frames", if the user selects "Forward", what should be exported? The same as if the user selected the parent tag with anidir "Reverse"?
Even without having child tags, just with one big tag (with reverse direction) that extends to the whole animation. What should be expected in each case?

NOTE: When testing this case, I found that there are some issues when exporting animations with nested tags.

Reply:
I believe that the best would be to not allow to select the animation direction when the play subtags is checked.

Reply:
The current v1.3 behavior should answer your questions. You will see that when "All frames" is selected, the "Forward" option is selected to indicate that we're going to playback all frames/tags as they are (the export must be exactly as the playback). If we change the "All frames" direction the final result will change.

But for the case of tags, the anidir indicates the direction of the tag: if we choose the parent tag and the anidir says "reverse", we are going to export the playing the tag with its original anidir, but if we change the anidir, we change the anidir of the parent (exported) tag.